bcgov/HOUS-permit-portal
# Add BCeID option when accepting invitation

## Description

Adds BCeID option when reviewer is accepting invitation. 

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

Implements [sc67729](https://app.shortcut.com/codechallengell/story/67729/add-bceid-login-to-reviewer-invitation-flow) / [BPHH-242](https://hous-bssb.atlassian.net/browse/BPHH-242)

## Steps to QA

1. Login as a super admin or review manager
2. Invite a reviewer / review manager to a jurisdiction
3. Open the invite link in an incognito window
4. See the option to login with BCeID - this should take the user to the BCeID login page and upon successful authentication, it should redirect back to the app and the user should be logged in.
5. If the BCeID is already associated with another account, the user will see an error message when redirected back to the app

## UI Accessibility Checklist

_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

- [ ] Update SSO Integration to support test / prod environments (pending QA / PROD environment setup)
- [ ] Add ENV variables as per SSO client details once configured
